third_party/maya/lib/usdMaya/userExportedAttributesUI.py

```diff
@@ -651,7 +658,7 @@ def __init__(self, parent=None):
         self.setAcceptDrops(True)
 
         self.setItemDelegateForColumn(ExportedAttributesModel.USD_ATTR_TYPE_COLUMN,
-            ExportedAttributesViewItemDelegate(USD_ATTR_TYPE_OPTIONS, self))
+            ExportedAttributesViewItemDelegate(USD_ATTR_TYPE_OPTIONS + ExportedAttribute.GetUserAttributeWriters(), self))
```

Comment:
not sure what Pixar's line length policy is, but 79 characters is standard for python.  line break before `USD_ATTR_TYPE_OPTIONS`.